extensions/chromium/claude-api-handler.js

```
@@ -0,0 +1,60 @@
+// Background script for handling Claude API calls
+// This solves CORS issues by making requests from the extension's background context
+
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
```

ESLint

[error] Expected to return a value at the end of arrow function.

(consistent-return)